frontend/apps/frontend/src/app/home/home/home.component.ts

```diff
@@ -1,14 +1,14 @@
-import { Component, OnInit } from "@angular/core";
+import {Component, OnInit} from "@angular/core";
 import {
   IDTokenPayload,
   MainframeIdTokenService,
   MainframeNavService,
-  PluginListService,
   Plugin,
+  PluginListService,
 } from "mainframe-connector";
-import { setTagColors } from "../../shared/classes/tagColors";
-import { NavBarService } from "../../core/nav-bar.service";
-import { SearchService } from "../../core/search.service";
+import {setTagColors} from "libs/core-ui/src/lib/syno-tag/tagColors";
+import {NavBarService} from "../../core/nav-bar.service";
+import {SelectOption} from "../../../../../../libs/core-ui/src/lib/syno-select-multiple/syno-select-multiple.component";
```

Comment:
illegal, des wa des ziel
```suggestion
import {SelectOption} from "core-ui";
```
keat in https://github.com/htl-grieskirchen-core/core/blob/develop/frontend/libs/core-ui/src/index.ts nu eitrogn damit do wos geht

Reply:
wo ma dabei sand de zeile 9 is a gonz sche cursed